packages/rancher-gatekeeper/rancher-gatekeeper.patch

```diff
@@ -259,28 +208,30 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper/charts-original/tem
 +          def_msg := sprintf("Label <%v: %v> does not satisfy allowed regex: %v", [key, value, expected.allowedRegex])
 +          msg := get_message(input.parameters, def_msg)
 +        }
-\ No newline at end of file
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper/charts-original/values.yaml packages/rancher-gatekeeper/charts/values.yaml
 --- packages/rancher-gatekeeper/charts-original/values.yaml
 +++ packages/rancher-gatekeeper/charts/values.yaml
-@@ -1,12 +1,12 @@
- replicas: 1
+@@ -1,15 +1,15 @@
```

Comment:
Why is replicas defaults to 3?

Reply:
@StrongMonkey The opa chart went through big changes in version `v3.1.0-beta.9`. 
This is being picked up from those changes after updating to `v3.1.0-beta.11`.

Reply:
That change is being picked up from updating to v3.1.0-beta.11 since there were big changes in the upstream v3.1.0-beta.9 we currently don't have.